app/controllers/ems_common.rb

```diff
@@ -437,6 +437,28 @@ def button
         end
         return
       end
+      if params[:pressed] == "ems_cloud_recheck_auth_status"
+        @record = find_by_id_filtered(model, params[:id])
+        authentications = @record.authentication_for_summary
+        authentications.collect do |auth|
+          result, details = @record.authentication_check_queue(auth[:authtype], :save => true)
+          label =
+              case auth[:authtype]
+                when "default" then _("Default")
+                when "metrics" then _("C & U Database")
+                when "amqp"    then _("AMQP")
+                else;               _("<Unknown>")
```

Comment:
Since I plan to use `authentication_check_types_queue`, I would not be rendering the flash message for each individual authentication type now...meaning the above code can be cleaned out and only a single flash message would be displayed -
```
Re-checking Authentication status initiated for the selected Cloud Provider
```